EarlGrey/Action/GREYActions.m

```diff
@@ -261,6 +254,69 @@ @implementation GREYActions
 
 #pragma mark - Private
 
+
+/**
+ *  Get WebView input text value.
+ *  @param element The element to target
+ */
++ (NSString *)grey_webviewInputValue:(id)element {
```

Comment:
Lets rename this to grey_webViewInputClearAndGetValue or something similar in tune with what it does.